js/embeddings.test.ts

```diff
@@ -50,4 +50,4 @@ test("Embeddings Test", async () => {
       expect(result.score).toBeLessThan(0.5);
     }
   }
-});
+}, 600000);
```

Comment:
10 minutes seems like a pretty long timeout for a unit test?

Reply:
This is the same as others -- it is just when the openai cache misses